### PR TITLE
Add lock-related endpoints

### PR DESCRIPTION
**Pull request by dragon-dxw**
New endpoints: DELETE /lock and PATCH /judgment
Add testing for all current endpoints
Use the new error message support in the api-client

### CODE REVIEW

**Comment by dragon-dxw** [comment]
[Implement checkin-and-update endpoint](https://trello.com/c/rX4KEuAC/262-implement-checkin-and-update-endpoint)

File: setup.cfg [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [flake8]
 max-line-length = 120
 exclude = .tox,.git,*/migrations/*,*/static/CACHE/*,docs,node_modules,venv
-ignore = I001, I002, I003, I004
```
Comment (lozette) [inline setup.cfg:4]: Can you update the commit message to indicate what you're excluding here

File: src/openapi_server/apis/writing_api.py [hunk 2]
```diff
@@ -165,7 +165,6 @@ async def judgment_uri_patch(
 
     client = client_for_basic_auth(token_basic)
     bytes_body = await request.body()
-    breakpoint()
```
Comment (lozette) [inline src/openapi_server/apis/writing_api.py]: Can you meld this one into 9ce875022d8d917a0a65e81d4ef808c4534dc351

File: setup.cfg (hunk 1 above)
Comment (dragon-dxw) [reply setup.cfg:4]: Done: mostly I was irritated at how isort gave no useful messages for diagnosing what was missing!

**Review by lozette — APPROVED** [review]

**Pull request merged**